### PR TITLE
Adding Libraries' branding materials via Makefile (includes ADR)

### PR DESCRIPTION
**Pull request by matt-bernhardt**
TL/DR: This adds a Makefile to the theme for importing some materials from our shared style library. Specifically, we are importing the main stylesheet and some images via this Makefile.

The scope of the Makefile, and some alternative structures for maintaining our brand, are the topic of the ADR that is part of this PR.

The ticket for this work is: https://mitlibraries.atlassian.net/browse/POST-16

---
In more detail, the commits in this PR break down as follows:
1. The first commit is the heart of this approach, which establishes the pathway for importing materials from the style library. It creates the Makefile (patterned after [our theme gem](https://github.com/MITLibraries/mitlibraries-theme/blob/main/Makefile)) which brings in whatever branding materials are going to be needed by the Omeka theme. Currently we import the stylesheet and three images (favicon, logo, and network graphic).

Materials imported via the Makefile are plaed as-is, without customization. Any needed adjustments to the stylesheet will be handled via local styles in this theme. Should we need to customize any images, they should be removed from the Makefile and managed entirely within the theme.

This commit includes some other minor adjustments - see the side effects section of that commit message for details.

2. The second commit contains the result of running the Makefile. it should be possible to check out this commit, run `make update`, and see no changes listed under `git status`. Running `make update` should only be necessary when there are upstream changes to our brand library which need to be picked up by the theme - so I'm choosing not to add a GH Actions step to require that it run cleanly on every PR. Because there isn't a version constraint in the Makefile, I want to keep this decoupled, to allow us to chose at the theme level when those assets need to be updated (This is also the same setup which the theme gem uses, so we are being consistent in this choice).

3. The third commit places the slim header and footer from the style library, into a files/common path. This commit makes no changes to these files, for clarity's sake. They will be customized, along with other needed changes, in the next commit.

4. With all the materials now imported or copy-pasted, the fourth commit makes the adjustments necessary for everything to function together. The header, footer, and layout templates are updated to run in Omeka, and refer to each other. Additionally, some SCSS files are adjusted to override values in the main stylesheet that are not valid in Omeka. **Please note** that the materials imported by the Makefile are not touched in this commit (as this would defeat the purpose of the Makefile and its update workflow). The details of these customizations are described in that commit message.

6. The final commit adds the result of the Sass compilation. Because the prior commits updates some SCSS files, we end up needing an updated `base.css` stylesheet. Again, see the full commit messages for the changes being made.

### CODE REVIEW

**Review by jazairi — APPROVED** [review]
Approving both the ADR and the code. Thanks for using multiple commits for this; it made it easier to follow.
